odxtools/basicstructure.py

```diff
@@ -22,12 +22,11 @@
 
 if TYPE_CHECKING:
     from .diaglayer import DiagLayer
-    from .endofpdufield import EndOfPduField
 
 
 @dataclass
 class BasicStructure(DopBase):
-    parameters: NamedItemList[Union[Parameter, "EndOfPduField"]]
+    parameters: NamedItemList[Parameter]
```

Comment:
Are you sure about this?

Reply:
yes: all fields are DOPs and thus encoded using `ValueParameter` and the code actually already works like this. I don't know how this sneaked, but I if I had to speculate, I would bet that it is an artifact from the time when EndOfPduFields were implemented...